src/components/Tabs.tsx

```diff
@@ -29,33 +29,40 @@ export interface TabsProps<V extends string, X extends Properties> {
  *
  * The caller is responsible for using `selected` / `onChange` to control
  * the current tab.
+ *
+ * If you want to tease apart Tabs from their TabContent, you can use the `Tab`
+ * and `TabContent` components directly.
  */
 export function TabsWithContent<V extends string, X extends Only<TabsContentXss, X>>(props: TabsProps<V, X>) {
-  const { selected, tabs, contentXss = {}, ...others } = props;
-  const selectedTab = tabs.find((tab) => tab.value === selected) || tabs[0];
-  const tid = useTestIds(others, "tab");
-
-  const onlyOneTabEnabled = tabs.filter((t) => !t.disabled).length === 1;
-
+  const onlyOneTabEnabled = props.tabs.filter((t) => !t.disabled).length === 1;
   return (
     <div>
       {!onlyOneTabEnabled && <Tabs {...props} />}
-      <div
-        aria-labelledby={`${selectedTab.value}-tab`}
-        id={`${selectedTab.value}-tabPanel`}
-        role="tabpanel"
-        tabIndex={0}
-        {...tid.panel}
-        css={{ ...Css.mt2.$, ...contentXss }}
-      >
-        {selectedTab.render()}
-      </div>
+      <TabContent {...props} />
+    </div>
+  );
+}
+
+export function TabContent<V extends string>(props: Omit<TabsProps<V, {}>, "onChange">) {
+  const tid = useTestIds(props, "tab");
+  const { selected, tabs, contentXss = {} } = props;
+  const selectedTab = tabs.find((tab) => tab.value === selected) || tabs[0];
+  return (
+    <div
+      aria-labelledby={`${selectedTab.value}-tab`}
+      id={`${selectedTab.value}-tabPanel`}
+      role="tabpanel"
+      tabIndex={0}
+      {...tid.panel}
+      css={{ ...Css.mt2.$, ...contentXss }}
```

Comment:
If we allow `contentXss` to supply more than just `Margin`, it might allow us to get the scrolling behavior of nested content within the tab panel as we want.